module5/lessons/Week2/WritingGoodDocumentation.md

```diff
@@ -0,0 +1,78 @@
+- Your README is how someone knows how to access your project (It should be written in a way that non-technical people can understand)
+- The README documents your experience and process to an employer (which is way more important than the project itself!)
+- Looking back at past READMEs is a great way to remind yourself of all you've done and prepare for interviews
+- **Taken cumulatively, your READMEs document your technical and professional growth**
+
+<section class="call-to-action" markdown="1">
+  <h3>Judging READMEs</h3>
+
+  In your breakout rooms, look through the READMEs below. We're going to go for first impressions here, so spend about 1 minute per README. Make some observations about the READMEs and add some stickies to [this Jamboard](https://jamboard.google.com/d/14atFSLLIqkl1Oa-EI5sfvS8EWaFH9xfpnceFP17mCHs/edit?usp=sharing)
+
+  README Links:
+  * [README #1](https://github.com/sertmer/check-yo-self)
+  * [README #2](https://github.com/Dustin-Mikusko/socialsentiment)
+  * [README #3](https://github.com/kaylagordon/ican2_FE)
+  * [README #4](https://github.com/turingschool-examples/overlook-api)
+  * [README #5](https://github.com/maddielaw/rec-me)
+  * [README #6](https://github.com/dcoleman21/Sweater-Weather)
+  * [README #7](https://github.com/me-md/me-md)
+</section>
+
+<section class="answer" markdown="1">
+### Some potential takeaways!
+
+- Too many images can make it too chaotic
+- When information isn't shared in a logical order, it's hard to follow
+- Deploy links and install instructions should be one of the first things you see
+</section>
+
+## The What
+
+So what should we include in a README? Well, it's not a one-size-fits-all thing. Depending on your project, what you're the most proud of, etc., you may want to highlight different things in your README. Here are some things we recommend including:
+- **Set Up Instructions**
+- **Deploy Link** (if applicable)
+- **Abstract/Elevator Pitch** (what problem is app trying to solve?)
+- **Contributors** (including who owned what pieces?)
+- **Links to Wireframe/Planning docs** (if applicable)
+- **Tech Stack** (including testing + accessibility)
+- **Context** (time allotted, # of people, mod, learning goals)
+- **Wins/Challenges** (personal or team)
+- **Future Additions** (these should be entered as Issues and on your Project Board as well)
+- **ONE gif** (show the highest impact feature of your app)
+- You can add comments in your README for things you want to document but don't want shown (just remember these are still viewable to everyone in the raw file)
+
+Obviously including all of this would be a bit much. Think about - What do you want someone to know about this project AND what do you want to remember about this project?
+
+## The How
+
+Markdown, the language of READMEs, can be tricky at first. [Here](https://about.gitlab.com/handbook/markdown-guide/) is a great guide to get you started. And as always, Google is your friend!

```

Comment:
This link is very overwhelming. I've used these cheat sheets in the past:

- https://www.markdownguide.org/cheat-sheet/
- https://www.markdownguide.org/extended-syntax

I like your link as a third option.

Reply:
Oooo your links are much better! Thanks for sharing them! I didn't have any links I've used in the past so I did a super quick google and found this one. But I agree it's a lot of content and pretty overwhelming.